### ISSUE TITLE
Add Docker host name to log

### ISSUE THREAD

**Issue by ezjacob-maheim** [sendgridlabs/loggly-docker#18]
Is there a way we can add docker hostname with the container hostname when pushing to loggly?

**Comment by jonathan-short** [comment]
@ezjacob-maheim sorry for the delayed response.  I haven't tested this, but I think you can use the --hostname option to docker run to format however you like.

**Comment by codezninja** [comment]
yeah that I know but the hostname for the container isn't added to the logs thats send to loggly. I was wondering if there was a way to do that
